### ISSUE TITLE
Generated JS files should be wrapped in an IIFE

### ISSUE THREAD

**Issue by meetandrew** [ericmdantas/generator-ng-fullstack#10]
I noticed that JS files are created aren't wrapped up in an IIFE.  Is there any reason for this?

**Comment by ericmdantas** [comment]
Hello!

That's in my todo list actually, I was about to commit it today, but I dont have internet at the moment.

I'll also move the "use strict" inside the functions, so it doesnt get mixed with others.

I might do something like:

```
;(function(angular)
{
"use strict";
// content goes here
}(window.angular));
```

**Comment by ericmdantas** [comment]
@meetandrew, just released v1.1.0. Mind to taking a look whenever possible? =]

**Comment by ericmdantas** [comment]
I'm closing this, let me know if anything goes wrong.
